plugins/sockbplog/sockbplog.c

```diff
@@ -56,6 +56,7 @@ static int handle_sockbplog_request_session(SBUF2 *sb, char *host)
     /* create a session/bplog */
     sess = osql_sess_create(sql, strlen(sql) + 1, tzname, type, rqid, uuid,
                             host, flags & OSQL_FLAGS_REORDER_ON);
+    free(sql);
```

Comment:
lets move this before return 0 to prevent double-free if the sess fails to be created.

Reply:
Yes. I overlooked it. Will do.